javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,90 @@
+// Shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function compareLength(arr) {
+  return arr.reduce((x, y) => (x.length <= y.length ? x : y));
+}
+console.log("The shortest word in array is " + compareLength(danishWords));
+// returns 'ø' << Added console log here instead of consolelogging inside func
+
+//Danish letter amount
+function countLetters(string) {
+  console.log(string);
+  const result = { total: 0, æ: 0, ø: 0, å: 0 };
+  [...string].forEach((element) => {
```

Comment:
Excellent use of the spread operator! So good! 

You probably know this, but another way to do this -- which might be a bit more readable -- would be `string.split("")`. But this is perfect